features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/components/APIDetailsTopMenu.jsx

```diff
@@ -110,6 +112,13 @@ const styles = (theme) => ({
     readOnlyStyle: {
         color: 'red',
     },
+    active: {
+        background: 'radial-gradient(#29bb89, #f7ea00)',
+        width: 8,
```

Comment:
please use `theme.spacing` better if we can replace with a `Box`

Reply:
Fixed